installation/linux.md

```diff
@@ -48,17 +48,17 @@ Alternatively resort to the [manual installation approach](linux.html#manual-ins
 Choose between the latest Beta release or a Snapshot with all incoming contributions, rebuild on [every change](https://oss.jfrog.org/webapp/#/builds/openHAB-Distribution).
 As openHAB 2 is still in an evolving state, the snapshot may be the **preferred choice**.
 
-The repository URLs will be stored in the file `/etc/apt/sources.list.d/openhab.list`.
+The repository URLs will be stored in the file `/etc/apt/sources.list.d/openhab2.list`.
 Be careful to not have conflicting repositories in your sources list.
```

Comment:
The only file I can think of right now, not (entirely) named openhab2 is `/var/log/openhab2/openhab.log`.

For the cleanup: depends on the setup. Here is what I would name the hammer method:

* backup config and userdata (icons etc)
* stop service
* purge
* delete
  * /opt/openhab
  * /etc/openhab
  * /usr/share/openhab
  * service file
  * logs
  * other known files by openhab1 or openhab2 if available 

I'm not sure about the exact path names.